ider/src/ider.component.spec.ts

```diff
@@ -28,9 +28,9 @@ describe('IderComponent', () => {
   })
 
   it('should correctly instantiate redirector and ider', () => {
-    component.mpsServer = 'testServer'
-    component.authToken = 'testToken'
-    component.deviceId = 'testDeviceId'
+    Object.defineProperty(component, 'mpsServer', { value: 'testServer' })
```

Comment:
this doesn't seem right to me, the component should already have the component defined.